android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java

```diff
@@ -400,6 +400,37 @@ static void trim(ReadableMap options, final Promise promise, ReactApplicationCon
     executeFfmpegCommand(cmd, tempFile.getPath(), ctx, promise, "Trim error", null);
   }
 
+  static void pad(ReadableMap options, final Promise promise, ReactApplicationContext ctx) {
+    String source = options.getString("source");
+    String width = options.getString("width");
+    String height = options.getString("height");
+
+    final File tempFile = createTempFile("mp4", promise, ctx);
+
+    ArrayList<String> cmd = new ArrayList<String>();
+    cmd.add("-y"); // NOTE: OVERWRITE OUTPUT FILE
+
+    // NOTE: INPUT FILE
+    cmd.add("-i");
+    cmd.add(source);
+
+    cmd.add("-vf");
+    cmd.add("scale="+width+":"+height+":force_original_aspect_ratio=decrease,pad="+width+":"+height+":(ow-iw)/2:(oh-ih)/2,setsar=1");
```

Comment:
In https://github.com/relivecc/react-native-video-processing/pull/4 we plan to update from 3 (this branch) to version 4. Make sure you check the scale parameter against that branch and/or version.

Reply:
Works! ✅

Reply:
Awesome 👍 Great job